openra/classes/file_location.py

```diff
@@ -18,6 +20,16 @@ def __init__(self, fs: FS, path: str, file: str):
         self.path = path
         self.file = file
 
+    def get_file_basename(self):
+        return re.sub(r"\.[^\.]+$", "", self.file)
```

Comment:
Why not `os.path.basename`?

Reply:
I'm not really sure why, but when I pass in test_file.txt, it's returning test_file.txt, instead of the expected test_file

Reply:
I'm not really sure what to do on these comments. I can get them to use os.path.splitext, and manually trim off the `.` for the ext. To be honest I have more faith in the regex personally though. But can update if needed.

Reply:
Ah ok I think I'm using "basename" wrong as it refers to the filename with the extension. I'll correct those function names.

I'll also get it to use that os splitext function as maybe that'll be more obvious to people that use python a lot than the regex

Reply:
Ok this is updated now